File: 1469-Minimum Number of Steps to Make Two Strings Anagram.py
class Solution:
    def minSteps(self, s: str, t: str) -> int:
        c = defaultdict(int)
        
        for i in s:
            c[i] += 1
        
        for i in t:
            c[i] -= 1
            
            
        res = 0
        #print(c)
        for i, v in c.items():
            res += max(0, v)
        return res
    
    # l e e t c o d e
    
            # # 
    # p r a c t i c e
    # x x x t c x x e
    

File: 1469-Minimum Number of Steps to Make Two Strings Anagram.cpp
class Solution {
public:
    int minSteps(string s, string t) {
        unordered_map<char, int> c;
        
        for(char i : s)
            c[i] += 1;
        for (char i : t)
            c[i] -= 1;
        
        int res = 0;
        for(auto &it : c){
            res += max(0, it.second);
        }
        
        return res;
    }
};